lib/public/views/lhcPeriods/ActiveColumns/lhcPeriodsActiveColumns.js

```diff
@@ -33,7 +33,12 @@ export const lhcPeriodsActiveColumns = {
                 { lhcPeriodName: name },
                 { class: `mh1 ${runsCount === 0 ? 'gray-darker' : ''}` },
             ),
-            frontLink('Data Passes', 'data-passes-per-lhc-period-overview', { lhcPeriodId: id }, { class: 'mh1' }),
+            frontLink(
+                ['Data Passes', h('.badge', `(${dataPassesCount})`)],
+                'data-passes-per-lhc-period-overview',
+                { lhcPeriodId: id },
+                { class: `mh1 ${dataPassesCount === 0 ? 'gray-darker' : ''}` },
```

Comment:
You have one of the two cases here:
If you display a link, it should look like a link. If you don't display a link when there is no data pass, it should not be a link.

On a side note: try to avoid as much as possible to use `mh`, it tends to add margin where you don't expect it, you should use `g` when you can (it may not apply here, it's more a general note)

Reply:
noted, I removed some `mh1` and utilised `g` and moved this format to separate file

Reply:
Where?

Reply:
ok, now, I removed margin and padding of name, but left `h('.mh4'),` between name and buttons, because `g` is not enough to get big enough gap

Reply:
Even g4?

Reply:
with `g4` and without `h('.mh4'),` you get equal gaps, also with some zoom level, name overlaps with `Runs` link

Reply:
anyway, whole conversation doesn't apply anymore